src/server/blockchain/__tests__/adminWallet.test.js

```diff
@@ -1,6 +1,6 @@
 import AdminWallet from '../AdminWallet'
 
-jest.setTimeout(30000)
+jest.setTimeout(3000000)
```

Comment:
@StanislavShevchenko Does it really need so much time to wait for the test to be done?

Reply:
[done](https://github.com/GoodDollar/GoodServer/pull/68/commits/1d02ca129b9addb3145466fe576deffbb5068ac0)